Makefile

```diff
@@ -61,7 +61,15 @@ $(DOCKER_DISTRIB):
 	curl -o $(DOCKER_DISTRIB) $(DOCKER_DISTRIB_URL)
 
 tests:
-	go test -cover -tags netgo ./...
+	echo "mode: count" > profile.cov
+	for dir in $$(find . -type f -name '*_test.go' | xargs -n1 dirname | sort | uniq); do \
```

Comment:
No -printf on mac os x I'm afraid; probably not important for this repo, but thats why its there.  Still want me to change it?